Makefile

```diff
@@ -9,12 +9,10 @@ test:
 	$(DCRUN) cargo test
 
 ## test-examples:	test example code using localstack
-# FIXME: turn this into an actual test with assertions
-# Currently you need to visually inspect that the correct log output is generated.
 test-examples:
 	./scripts/run_example.sh
-	sleep 20
-	./scripts/read_logs.sh
+	sleep 10
```

Comment:
I see that the first [CI run](https://github.com/harrison-ai/cobalt-aws/runs/7244920043) failed with "Failed to determine group name". It's gross that we need to sleep here, but setting this back to `20` may help get that passing in CI.